executor/slow_query.go

```diff
@@ -256,29 +301,122 @@ func (e *slowQueryRetriever) getBatchLog(reader *bufio.Reader, offset *offset, n
 					e.fileLine = 0
 					file := e.getNextFile()
 					if file == nil {
-						return log, nil
+						return combineLogs(logs, num), nil
```

Comment:
No need to call `combineLogs`? since it will do a useless memory copy. just return `[][]string{log}` will be ok in the old method?